vta/hardware/chisel/src/main/scala/core/TensorGemm.scala

```diff
@@ -116,7 +119,7 @@ class MatrixVectorCore(implicit p: Parameters) extends Module {
     val acc_o = new TensorClientData(tensorType = "acc")
     val out = new TensorClientData(tensorType = "out")
   })
-  val dot = Seq.fill(size)(Module(new DotProduct(dataBits, size)))
+  val dot = Seq.fill(size)(Module(new DotProduct(inpBits, wgtBits, size)))
```

Comment:
can we do `val dot = Seq.fill(size)(Module(new DotProduct(aBits = inpBits, bBits = wgtBits, size)))` instead?. I think the convention is that if the argument is does not have the same name, then you have to explicitly assigned it.